src/foam/core/Slot.js

```diff
@@ -403,6 +408,21 @@ foam.CLASS({
   ]
 });
 
+foam.CLASS({
+  package: 'foam.core.internal',
+  name: 'Not',
+
+  properties: [
+    'a',
+    {
+      name: 'output',
+      expression: function(a) {
+        return ! (!! a);
```

Comment:
Why not just !?

Reply:
too bad you can't just make the expression be on `a$not`